### DISCUSSION TITLE
How to running on Linux last release git extension?

### DISCUSSION THREAD

**Discussion by GermanAizek** [gitextensions/gitextensions#10322] [Q&A] [answered]
I have latest Mono version, but there are so many graphical bugs in it, some menus are not in usual places, text gets out of elements and bug with mouse focus click in area.

![image](https://user-images.githubusercontent.com/21138600/199767998-a130260e-f2f6-4e28-a5ad-7feb0f49382c.png)

**Comment by gerhardol** [comment, marked as answer]
See some discussions in https://github.com/gitextensions/gitextensions/issues/6051
